src/plugins/graph/hooks/use-plot.ts

```diff
@@ -149,7 +147,16 @@ export const usePlotResponders = (props: IPlotResponderProps) => {
       });
       return () => disposer();
     }
-  }, [dataset, callRefreshPointPositions, refreshPointSelection]);
+  }, [dataset, callRefreshPointPositions]);
+
+  // respond to selection change
+  useEffect(function respondToSelectionChange() {
+    const disposer = reaction(
+      () => [dataset?.selectedAttributeIdString, dataset?.selectedCaseIdString],
+      () => refreshPointSelection()
+    );
+    return () => disposer();
```

Comment:
FYI, this can be simplified to:
```suggestion
    return reaction(
      () => [dataset?.selectedAttributeIdString, dataset?.selectedCaseIdString],
      () => refreshPointSelection()
    );
```